opencensus/exporters/stats/stackdriver/internal/stackdriver_exporter.cc

```diff
@@ -118,7 +118,14 @@ bool StackdriverExporter::Handler::MaybeRegisterView(
     const opencensus::stats::ViewDescriptor& descriptor) {
   const auto& it = registered_descriptors_.find(descriptor.name());
   if (it != registered_descriptors_.end()) {
-    return it->second == descriptor;
+    if (it->second == descriptor) {
+      return true;
+    } else {
```

Comment:
Optional: can deindent this.

Reply:
Also reversed the order, so it follows the pattern of branching on error.